src/block/block_ckpt.c

```diff
@@ -482,9 +522,22 @@ __ckpt_add_blk_mods_alloc(
             continue;
 
         if (block->created_during_backup)
-            WT_RET(__ckpt_add_blkmod_entry(session, blk_mod, 0, block->allocsize));
+            WT_RET(__ckpt_mod_blkmod_entry(session, blk_mod, 0, block->allocsize, true));
+        else
+            /* Always set the bit for the header. */
+            WT_RET(__ckpt_mod_blkmod_entry(session, blk_mod, 0, 4096, true));
+
+        if (clear) {
+            /*
+             * First clear any bits from the discard list. Then add in anything in the allocation
+             * list.
+             */
+            WT_EXT_FOREACH (ext, ci->discard.off) {
```

Comment:
Hi @sueloverso , as with my comment above the Code Change Report indicates that the tests used for code coverage don't reach this `WT_EXT_FOREACH` loop. Are there tests that that indicate we're covering this?

Reply:
Both of these showed a bug in the PR and both have been fixed with the real change.